components/pill/w-pill.vue

```diff
@@ -22,16 +23,28 @@ const closeClasses = computed(() => [
   ccPill.close,
   p.suggestion ? ccPill.suggestion : ccPill.filter
 ]);
+
+const openFilterSrText = i18n._({
+  id: 'pill.aria.openFilter',
+  message: 'Open filter CCC',
+  comment: 'Fallback screenreader message for open filter'
+});
+const removeFilterSrText = i18n._({
+  id: 'pill.aria.removeFilter',
+  message: 'Remove filter {label} CC',
```

Comment:
What's with the `CCC` 😄?

Reply:
Wuppps, I think this comes from when I was testing that I linked the project correctly! Will remove it :p